pkg/controller/util/node/controller_utils.go

```diff
@@ -29,7 +29,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Nit: I personally don't care, but I think some people have opinions on the `v1 ".../v1"` vs `".../v1` debate - was this change intentional/do you know if k8s has an overall policy?

Reply:
it's changed by vscode automatically :smile: